src/services/ShareService.js

```diff
@@ -242,7 +246,31 @@ export class ShareService {
 			topics: { current }
 		} = state;
 
-		extractedState[QueryParameters.TOPIC] = current;
+		if (current) {
+			extractedState[QueryParameters.TOPIC] = current;
+		}
+
+		return extractedState;
+	}
+	/**
+	 * @private
+	 * @returns {object} extractedState
+	 */
+	_extractCatalogNodes() {
+		const { StoreService: storeService } = $injector.inject('StoreService');
+
+		const state = storeService.getStore().getState();
+		const extractedState = {};
+
+		//current topic
+		const {
+			catalog: { openNodes }
+		} = state;
+
+		if (openNodes.length > 0) {
+			extractedState[QueryParameters.CATALOG_NODE_IDS] = openNodes.join();
+		}
+
 		return extractedState;
 	}
```

Comment:
`_extractCatalogNodes` returns  the content as a single (implicit joined) string, if multiple nodes are available.
Other methods like `_extractLayers` already returning the content as an array, if the content is potentially a multiple.

Is there any technical reason to change from the used pattern (_**methodNameWithPlural**_ -> returns Array)?

Reply:
Thx, fixed it